packages/actions/src/player-ui-actions.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  SET_CURRENT_WARGAME_PLAYER,
+  // SET_FORCE,
```

Comment:
@lilitkarapetyan - for these commented out actions, are you moving them over one by one?